samples/csharp/getting-started/console-teleprompter/README.md

```diff
@@ -1,7 +1,7 @@
 C# Console Application Sample
 ================
 
-This sample is created during the [Console Application Tutorial](https://docs.microsoft.com/dotnet/articles/csharp/tutorials/console-teleprompter)
+This sample is created during the [Console Application Tutorial](https://docs.microsoft.com/dotnet/csharp/tutorials/console-teleprompter)
```

Comment:
I don't think that link text is a proper noun ???? ...

*console application tutorial*